scripts/completion/dvc.bash

```diff
@@ -82,7 +82,8 @@ _dvc () {
       *)  COMPREPLY=($(compgen -W "$_dvc_commands" -- "$word")) ;;
     esac
   elif [ "${COMP_CWORD}" -eq 2 ]; then
-    local options_list="_dvc_${COMP_WORDS[1]}"
+    local replace_hyphen=$(echo ${COMP_WORDS[1]} | sed 's/-/_/g')
```

Comment:
Let's move replace_hyphen into a nested function and use it here and in `-eq 3` block down below to replace hyphen for `COMP_WORDS[1]` and `COMP_WORDS[2]`. That way we will fix both issues with `-v` + TAB that we've discussed above.

Reply:
@efiop 
```
elif [ "${COMP_CWORD}" -eq 2 ]; then
    if [[ ${COMP_WORDS[1]:0:1} == "-" ]]; then
      return 0
    else
      local replace_hyphen=$(echo ${COMP_WORDS[1]} | sed 's/-/_/g')
      local options_list="_dvc_${replace_hyphen}"

      COMPREPLY=($(compgen -W "$_dvc_global_options ${!options_list}" -- "$word"))
    fi
  elif [ "${COMP_CWORD}" -eq 3 ]; then
    if [[ ${COMP_WORDS[2]:0:1} == "-" ]]; then
      return 0
    else
      local replace_hyphen=$(echo ${COMP_WORDS[2]} | sed 's/-/_/g')
      local options_list="_dvc_${COMP_WORDS[1]}_${COMP_WORDS[2]}"

      COMPREPLY=($(compgen -W "$_dvc_global_options ${!options_list}" -- "$word"))
    fi
  fi
```

We can also add that if condition which checks if it is a flag and then stops autocomplete. Will this work?

Reply:
Well, it will, but why add that check? The issue here is that `_dvc_-v` is not a valid var name. If we simply replace `-` with `_` we will get `_dvc__v`, shich is always empty, so everything will just start working automatically.